src/main/java/com/amazon/dlic/auth/http/jwt/AbstractHTTPJwtAuthenticator.java

```diff
@@ -46,17 +48,20 @@
     private final static Logger log = LogManager.getLogger(AbstractHTTPJwtAuthenticator.class);
 
     private static final String BEARER = "bearer ";
+    private static final Pattern BASIC = Pattern.compile("^\\s*Basic\\s.*", Pattern.CASE_INSENSITIVE);
```

Comment:
Factor out BASIC pattern as it is re-used ?

Reply:
Out of scope for PR but any reason why HTTPJwtAuthenticator 
 does not inherit from AbstractHTTPJwtAuthenticator ?

Reply:
I don't see any good reason, will be good to cleanup this in a follow up PR.